### PR TITLE
[ACCEL] hides cohort management dashboard for learner roles

### PR DESCRIPTION
**Pull request by Golub-Sergey**
**Description:** hides cohort management dashboard tab and link for learner roles

**Youtrack:** [tiket](https://youtrack.raccoongang.com/issue/ACCEL-519), [review ticket](https://youtrack.raccoongang.com/issue/ACCEL-523)

### CODE REVIEW

File: lms/djangoapps/cohort_management_dashboard/tab.py [hunk 1]
```diff
@@ -11,5 +12,9 @@ class CohortManagementDashboardTab(EnrolledTab):
 
     @classmethod
     def is_enabled(cls, course, user=None):
-        return super(CohortManagementDashboardTab, cls).is_enabled(course, user)
+        if (
+            has_access(user, 'staff', course, course.id) or has_access(user, 'instructor', course, course.id)
+            or user.is_staff or user.is_superuser
+        ):
+            return super(CohortManagementDashboardTab, cls).is_enabled(course, user)
```
Comment (idegtiarov) [inline lms/djangoapps/cohort_management_dashboard/tab.py]: Does None is expected value or method should return boolean?

Comment (Golub-Sergey) [reply lms/djangoapps/cohort_management_dashboard/tab.py]: method should return boolean, this line returns True

Comment (idegtiarov) [reply lms/djangoapps/cohort_management_dashboard/tab.py]: Ok, great! But if condition false method returns None.

Comment (Golub-Sergey) [reply lms/djangoapps/cohort_management_dashboard/tab.py]: done

**Pull request merged**